src/app/shared/model/series.model.ts

```diff
@@ -120,6 +120,9 @@ export class SeriesModel extends BaseModel implements HasUpload {
   }
 
   discard(): any {
+    if (this.changed('accountId')) {
+      this.set('accountId', this.original['accountId']);
+    }
     super.discard();
```

Comment:
I found that BaseModel discard was clearing the accountId since it's a sort of pseudo field that comes from the parent account.